<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerKinectTrackingModuleWidget</class>
 <widget class="QWidget" name="qSlicerKinectTrackingModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ConnectorPropertyFrame">
     <property name="title">
      <string/>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>52</y>
        <width>490</width>
        <height>201</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ConnectorHostAddressLabel">
           <property name="text">
            <string>Host name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorHostAddressEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ConnectorPortLabel">
           <property name="text">
            <string>  Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorPortEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="ConnectorHostAddressLabel_2">
           <property name="text">
            <string>Robot IP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorHostAddressEdit_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ConnectorPortLabel_2">
           <property name="text">
            <string>  Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorPortEdit_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="FrequencyLabel">
             <property name="text">
              <string>Frequency:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="FrameFrequency">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhDigitsOnly</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="UseCompressLabel">
             <property name="text">
              <string>Use Compress:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="UseCompressCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Active</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ConnectorStateCheckBox">
             <property name="minimumSize">
              <size>
               <width>19</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Start Video</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="StartVideoCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="NodeSelectorLabel">
         <property name="text">
          <string>TargetModelInputs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="qMRMLNodeComboBox" name="NodeSelector" native="true">
         <property name="noneEnabled" stdset="0">
          <bool>true</bool>
         </property>
         <property name="nodeTypes" stdset="0">
          <stringlist>
           <string>vtkMRMLModelNode</string>
          </stringlist>
         </property>
         <property name="addEnabled" stdset="0">
          <bool>true</bool>
         </property>
         <property name="removeEnabled" stdset="0">
          <bool>true</bool>
         </property>
         <property name="showChildNodeTypes" stdset="0">
          <bool>true</bool>
         </property>
         <property name="selectNodeUponCreation" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="NodeSelectorLabel2">
         <property name="text">
          <string>TransformationRobotToSlicer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="qMRMLNodeComboBox" name="NodeSelectorTransform" native="true">
         <property name="nodeTypes" stdset="0">
          <stringlist>
           <string>vtkMRMLLinearTransformNode</string>
           <string>vtkMRMLBSplineTransformNode</string>
           <string>vtkMRMLGridTransformNode</string>
           <string>vtkMRMLTransformNode</string>
          </stringlist>
         </property>
         <property name="renameEnabled" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="SlicerVideoWidget" name="graphicsView" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>SlicerVideoWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/SlicerVideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ConnectorStateCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerKinectTrackingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>NodeSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerKinectTrackingModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>NodeSelectorTransform</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
